llvm/test/tools/sycl-post-link/sycl-bfloat16-imported-symbols.ll

```diff
@@ -0,0 +1,64 @@
+; This test checks that sycl-post-link can correctly analyze device code to
+; detect bfloat16 devicelib functions are used and add these used functions
+; too imported symbol list.
+
+; RUN: sycl-post-link %s -emit-param-info -symbols -emit-imported-symbols -properties -split=auto -o %t.table
+; RUN: FileCheck %s -input-file=%t_0.prop --check-prefix=CHECK-BF16
+
+; CHECK-BF16: [SYCL/imported symbols]
+; CHECK-BF16-NEXT: __devicelib_ConvertFToBF16INTEL
+
+%"class.sycl::_V1::id" = type { %"class.sycl::_V1::detail::array" }
```

Comment:
Please, remove all unnecessary instructions, attributes and metadata.

Reply:
Hi, @maksimsab 
Done.
Thanks very much!